html/changelogs/revreflavor.yml

```diff
@@ -0,0 +1,44 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: chada
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "Replaced the 'Revolutionaries' with 'The Contendors', a more neutral term that will hopefully make it easier for them to recruit in the Crew."
+  - tweak: "Replaced the 'Loyalists' with 'The Fellowship', which should leave them as the more friendly sounding of the two yet still be much less pro-Command."
+  - imageadd: "Added two brand new hud icons with these differences in mind. Made better ones afterall and had to change this entry."
+  - tweak: "Gave the contendors and fellowship more equal footing."
```

Comment:
Very undescriptive CL entry. What exactly did you do here? Add it.

Reply:
I think the new entry should explain it in more detail unless you meant all of it and not just the last entry.

Reply:
If the new entry doesn't solve this, please re-open this and I'll make more changes.